Showcase-iOS/Dashboard/View/HomeViewController.swift

```diff
@@ -79,5 +81,23 @@ extension HomeViewController: HomePresenterViewable {
     }
 }
 
-
-
+extension HomeViewController: UISearchResultsUpdating, UISearchControllerDelegate {
+   
+    func updateSearchResults(for searchController: UISearchController) {
+        guard let text = searchController.searchBar.text else {
+            return
+        }
+        if text.count > 0 {
```

Comment:
@edward-mtsh This logic should be in the Presenter and it needs to be unit tested please. All the of filtering code.